use error_handling::Guess;

fn main() {
    let test = Guess { value: 4 };

    println!("{:#?}", test);
}
